.github/workflows/test-all-packages.yml

```diff
@@ -45,6 +45,7 @@ jobs:
   # We split the package tests into two jobs because type linting
   # is inefficient and slow https://github.com/typescript-eslint/typescript-eslint/issues/2094
   lint-primary:
+    timeout-minutes: 10
```

Comment:
looks like this one is usually under 5min but better to have its timeout match the other one